docs/osquery/osquery-response-action.asciidoc

```diff
@@ -33,7 +33,9 @@ You can add Osquery Response Actions to new or existing custom query rules. Quer
 + 
 NOTE: If the rule's investigation guide is using an Osquery query, you'll be asked if you want to add the query as an Osquery Response Action. Click *Add* to add the investigation guide's query to the rule's Osquery Response Action. 
 . Specify whether you want to set up a single live query or a pack:
-** *Query*: Select a saved query or enter a new one. After you enter the query, you can expand the **Advanced** section to view or set {kibana-ref}/osquery.html#osquery-map-fields[mapped ECS fields] included in the results from the live query. Mapping ECS fields is optional.
+** *Query*: Select a saved query or enter a new one. After you enter the query, you can expand the **Advanced** section to set a timeout period for the query, and view or set {kibana-ref}/osquery.html#osquery-map-fields[mapped ECS fields] included in the results from the live query (optional). 
```

Comment:
```suggestion
** *Query*: Select a saved query or enter a new one. After you enter the query, you can expand the **Advanced** section to set a timeout period for the query, and view or set {kibana-ref}/osquery.html#osquery-map-fields[mapped ECS fields] to include in the results from the live query (optional). 
```
For your consideration

Reply:
I'm leaning towards keeping this as is because the fields are already pre-populated in this section of the form. Users can choose to include more fields or remove some, but for the most part, they don't need to do a whole lot with the field mappings.